### PR TITLE
Update Codespace Instructions

### PR DESCRIPTION
**Pull request by BrightTheBackpack**
## Summary of the problem
Clarifies some steps of setting up a dev environment

## Describe your changes

Just some additions based on the slack thread

### CODE REVIEW

**Comment by garyhtou** [comment]
Thanks! I've asked @manuthecoder to leave a review on this PR since he regularly uses Codespaces!

**Review by manuthecoder — DISMISSED** [review]
Mostly LGTM! Here's a couple points of feedback...

> and open it locally using ctrl+shift+p (if that does nothing, click on the search bar at the top of your codespace, then click Show and Run Commands >, then search Codespaces: Open in VS Code Desktop.

You could probably replace this with "open the command palette [...] search Codespaces: Open in VS Code Desktop" 

<br />

> hcb does not work on the online version

Maybe replacing "online" with "web" would be clearer?

<br />

Also I'm not sure if the github codespaces correctly runs the database seed upon setup. If it doesn't, make sure to tell users to run  rake `db:seed`

<br />


> Login using the email admin@bank.engineering

A little outside of this PR's scope, but it might be useful to move this to another section, since this information isn't specific to just GitHub Codespaces (maybe a section called "Development credentials" or something like that)

<br />


Use this markdown to create a nice button
```md
[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=135250235&skip_quickstart=true&machine=premiumLinux&devcontainer_path=.devcontainer%2Fdevcontainer.json&geo=UsWest)
```

This selects an 8 core machine by default and formats it into a nice badge, see preview below:

[Open in GitHub Codespaces](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=135250235&skip_quickstart=true&machine=premiumLinux&devcontainer_path=.devcontainer%2Fdevcontainer.json&geo=UsWest)

**Comment by BrightTheBackpack** [comment]
Done! Database seeding is already done when the Codespace is opened.

File: README.md [hunk 1]
```diff
@@ -42,9 +42,11 @@ Please email [hcb-security@hackclub.com](mailto:hcb-security@hackclub.com) to re
...
+
 [GitHub Codespaces](https://docs.github.com/en/codespaces) allows you to run a development environment without installing anything on your computer, allows for multiple instances, creates an overall streamlined and reproducible environment, and enables anyone with browser or VS Code access to contribute.
 
-To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart) and run `bin/dev`. If you can't open the link that is printed in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal.
+To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart), open the command palette(`ctrl+shift+p`), and search `Codespaces: Open in VS Code Desktop`. HCB does not work on the web version of Codespaces. Run `bin/dev`. If you can't open the link that is printed in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal.
```
Comment (sampoder) [inline README.md]:
```suggestion
To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart), open the command palette(<kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>P</kbd>), and search `Codespaces: Open in VS Code Desktop`. HCB does not work on the web version of Codespaces. Run `bin/dev`. If you can't open the link that is printed in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal. 
```

File: README.md [hunk 2]
```diff
@@ -68,6 +70,10 @@ Then, to start the development server:
 
 If you have more questions about development check out our [wiki](https://github.com/hackclub/hcb/wiki)
 
+### Development Credentials
```
Comment (sampoder) [inline README.md]:
```suggestion
### Development Account
```

File: README.md [hunk 3]
```diff
@@ -42,9 +42,11 @@ Please email [hcb-security@hackclub.com](mailto:hcb-security@hackclub.com) to re
...
+
 [GitHub Codespaces](https://docs.github.com/en/codespaces) allows you to run a development environment without installing anything on your computer, allows for multiple instances, creates an overall streamlined and reproducible environment, and enables anyone with browser or VS Code access to contribute.
 
-To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart) and run `bin/dev`. If you can't open the link that is printed in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal.
+To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart), open the command palette(<kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>P</kbd>), and search `Codespaces: Open in VS Code Desktop`. HCB does not work on the web version of Codespaces. Run `bin/dev`. If you can't open the link that is printed in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal.
```
Comment (sampoder) [inline README.md]:
```suggestion
To get started, [whip up a codespace](https://docs.github.com/en/codespaces/getting-started/quickstart), open the command palette(<kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>P</kbd>), and search `Codespaces: Open in VS Code Desktop`. HCB does not work on the web version of Codespaces. Run `bin/dev`. If you can't open the link that is printed in the terminal, ensure the `3000` port is public under the `PORTS` tab in your terminal.
```

File: README.md [hunk 4]
```diff
@@ -68,6 +70,10 @@ Then, to start the development server:
 
 If you have more questions about development check out our [wiki](https://github.com/hackclub/hcb/wiki)
 
+### Development Account
+
+Login using the email admin@bank.engineering. There'll be a button to open a virtual inbox to get the login code
```
Comment (sampoder) [inline README.md]:
```suggestion
Login using the email admin@bank.engineering. Use Doorkeeper (`/doorkeeper`) to access the development email outbox and retrieve the login code.
```

**Review by sampoder — APPROVED** [review]
LGTM! Thank you Bright & Manu.

**Pull request merged**